bin/generate-groups-inventory.bash

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+set -eu -o pipefail
+
+
+here="$(dirname "$(readlink -f "$0")")"
+# shellcheck source=bin/common.bash
+source "${here}/common.bash"
+
+# shellcheck source=bin/common.bash
+source "${here}/inventory-parser.bash"
+
+# shellcheck disable=SC2154
+if [ -e "${config[groups_inventory_file]}" ]; then
+    log_info_no_newline "${config[groups_inventory_file]} will be overwritten, Proceed [y/N] ? "
+    read -n 1 -r reply
+    if [[ "${reply,,}" != "y" ]]; then
+        exit 1
+    fi
+fi
+
+if [[ "$(group_exists "${config[inventory_file]}" all)" == "true" ]]; then
+    all_section="$(get_section "${config[inventory_file]}" all)"
+    echo -e "$all_section\n" > "${config[groups_inventory_file]}"
+else
+    log_error "Error: [all] group is not defined in ${config[inventory_file]}"
```

Comment:
Is this fatal enough to need to exit?

```suggestion
    log_error "Error: [all] group is not defined in ${config[inventory_file]}"
    exit 1
```

Reply:
Not sure to be honest, but I suppose kubespray relies on all to be defined for its playbooks

Reply:
So as it is right now it isn't exiting. So my question was (I see that my earlier question was confusing) if you think we should exit instead of leaving as it is right now and not exit.
But then if I hear you correctly you think that we should exit